src/blueprints/tv2/factories/tv2-sisyfos-audio-timeline-object-factory.ts

```diff
@@ -0,0 +1,37 @@
+import { Tv2AudioTimelineObjectFactory } from '../value-objects/factories/tv2-audio-timeline-object-factory'
+import { SisyfosChannelsTimelineObject, SisyfosType } from '../../timeline-state-resolver-types/sisyfos-types'
+import { Tv2SisyfosLayer } from '../value-objects/tv2-layers'
+import { DeviceType } from '../../../model/enums/device-type'
+import { Tv2BlueprintConfiguration } from '../value-objects/tv2-blueprint-configuration'
+
+export class Tv2SisyfosAudioTimelineObjectFactory implements Tv2AudioTimelineObjectFactory {
+  public createMicrophoneDownTimelineObject(blueprintConfiguration: Tv2BlueprintConfiguration): SisyfosChannelsTimelineObject {
+    return this.createMicrophoneTimelineObject(blueprintConfiguration, 0)
+  }
+
+  public createMicrophoneUpTimelineObject(blueprintConfiguration: Tv2BlueprintConfiguration): SisyfosChannelsTimelineObject {
+    return this.createMicrophoneTimelineObject(blueprintConfiguration, 1)
+  }
+
+  // Todo: Figure out what 'isPmg' stands for and write full name instead of abbreviation
+  private createMicrophoneTimelineObject(blueprintConfiguration: Tv2BlueprintConfiguration, isPmg:  0 | 1 | 2): SisyfosChannelsTimelineObject {
```

Comment:
It's `isPgm` and not `isPmg` :) 

If this is for Sisyfos, then it's whether the audio fader is active or not.

Reply:
I believe the values means:
- 0 - Off
- 1 - On
- 2 - VoiceOver

Reply:
Would be nice if was extracted into an enum.
For now that enum can just live privately inside this file

Reply:
Fixed typo, and changed type to an enum.